packages/big-design/src/components/StatefulTable/spec.tsx

```diff
@@ -124,9 +124,9 @@ test('renders rows without checkboxes when opting in to selectable', () => {
 
 test('items are unselected by default', () => {
   const { container } = render(getSimpleTable({ selectable: true }));
-  const checkbox = container.querySelector('tbody > tr input') as HTMLInputElement;
+  const checkboxes: NodeListOf<HTMLInputElement> = container.querySelectorAll('tbody > tr input');
```

Comment:
Could we use `findByRole` instead?

Reply:
<img width="939" alt="Screen Shot 2022-06-14 at 2 40 55 PM" src="https://user-images.githubusercontent.com/99345561/173674690-f1a747bc-626e-40a6-85c9-8e61fedcee09.png">